### ISSUE TITLE
Update Spark to version 1.5.0

### ISSUE THREAD

**Issue by paulochf** [jupyter/docker-stacks#26]
Just passing by to say that, today, I tried to use all-spark-notebook for Spark 1.5.0 and it got errors. I guess the jar repositories don't have all the packages yet.

Got some "not found" during build, like

```
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  ::              FAILED DOWNLOADS            ::
[warn]  :: ^ see resolution messages for details  ^ ::
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  :: org.apache.spark#spark-core_2.10;1.5.0!spark-core_2.10.jar
[warn]  :: org.apache.spark#spark-launcher_2.10;1.5.0!spark-launcher_2.10.jar
[warn]  :: org.apache.spark#spark-network-common_2.10;1.5.0!spark-network-common_2.10.jar
[warn]  :: org.apache.spark#spark-network-shuffle_2.10;1.5.0!spark-network-shuffle_2.10.jar
[warn]  :: org.apache.spark#spark-unsafe_2.10;1.5.0!spark-unsafe_2.10.jar
[warn]  :: org.apache.spark#spark-streaming_2.10;1.5.0!spark-streaming_2.10.jar
[warn]  :: org.apache.spark#spark-sql_2.10;1.5.0!spark-sql_2.10.jar
[warn]  :: org.apache.spark#spark-catalyst_2.10;1.5.0!spark-catalyst_2.10.jar
[warn]  :: org.apache.spark#spark-mllib_2.10;1.5.0!spark-mllib_2.10.jar
[warn]  :: org.apache.spark#spark-graphx_2.10;1.5.0!spark-graphx_2.10.jar
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
```

Let's watch [the packages repo](https://jcenter.bintray.com/org/apache/spark/) to see when they become available.

**Comment by parente** [comment]
Yep. And the other thing that still needs solving is how to tag the current image so that folks that still want to use 1.4.1 aren't surprised the next time they docker pull (Issue #12)

**Comment by parente** [comment]
Closed by #56. Build is running and will be on Docker Hub when ready.
